src/ir_LG.cpp

```diff
@@ -315,9 +327,8 @@ lg_ac_remote_model_t IRLgAc::getModel(void) const {
 /// @return true, if it is AKB74955603 message. Otherwise, false.
 /// @note Internal use only.
 bool IRLgAc::_isAKB74955603(void) const {
-  return (((_.raw & kLgAcAKB74955603DetectionMask) && _isNormal()) ||
-          isSwingV() ||
-          isLightToggle());
+  return ((_.raw & kLgAcAKB74955603DetectionMask) && _isNormal()) ||
+      isSwingV() || isLightToggle();
```

Comment:
My head is confused but verify indentation here.

Reply:
IIRC the "rules" according to the style guide are when breaking a line, indent to inside the enclosing parenthesis, if it exists. e.g. The previous commit has had the extra parenthesis/brackets. e.g. https://google.github.io/styleguide/cppguide.html#Boolean_Expressions

Without an enclosing parenthesis (and it doesn't fit), the default is to indent by 4 spaces (from the current indent level).
See:
* https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions
* https://google.github.io/styleguide/cppguide.html#Function_Calls
* https://google.github.io/styleguide/cppguide.html#Braced_Initializer_List_Format

Reading it again ... it's a murky area. Happy to put back the parenthesis .

Reply:
No this is better, I just don't have the rules in my head in regards to multiline functions returns etc etc ;)

Reply:
The rules are vague on this particular point. So you're not alone